cmd/restic/secondary_repo.go

```diff
@@ -29,6 +29,7 @@ func fillSecondaryGlobalOpts(opts secondaryRepoOptions, gopts GlobalOptions, rep
 	var err error
 	dstGopts := gopts
 	dstGopts.Repo = opts.Repo
+	dstGopts.RepositoryFile = ""
```

Comment:
I think it would be more useful to support `repository-file2` in `initSecondaryRepoOptions`.

Reply:
I will look in to that.
Yes, that would allow to use repositories files for both dst and src. However, that would not directly fix a combination of `--repo2` and `--repository-file`. That was atleast the reason for me to make this a bugfix. And later add a PR with repository-file2 as a feature.